invenio_administration/assets/semantic-ui/js/invenio_administration/src/components/Formatter.js

```diff
@@ -18,8 +18,7 @@ const elementTypeMap = {
 class Formatter extends React.Component {
   render() {
     const { resourceSchema, result, property } = this.props;
-    const resourceSchemaProperty = property.replace(/\./g, ".properties.");
-    const typePath = `${resourceSchemaProperty}.type`;
+    const typePath = `${property}.type`;
```

Comment:
Does this work with a nested properties? I remember that I added the parsing because in the `property` value we do not have the same structure as in the resourceSchema, we need to replace the dots with the `properties` as done in the resourceSchema to be able to get the correct Element, unless I am missing something.